eth/contracts/Most.sol

```diff
@@ -257,6 +264,9 @@ contract Most is
         return bytes32(uint256(uint160(addr)));
     }
 
+    // Some event like CommitteeUpdate might be Emitted here and in constructor
+    // For committee members to track their participation in comittee
+
```

Comment:
added - but hwy emit in the constructor ? it always starts with an id of 0 and there is nothing to listen to before the contract is initialized ?

Reply:
agree, constructor emit might be ommited.